pkg/netutil/netutil.go

```diff
@@ -0,0 +1,16 @@
+package netutil
+
+import (
+	"os/exec"
+	"strings"
+)
+
+func PublicIP() (string, error) {
+	cmd := exec.Command("curl", "-4", "ifconfig.me")
```

Comment:
There is no need to use curl if there is another way to do the same thing. Just use http.Client to get the public IP address.

Reply:
@eahydra done